misk/src/main/kotlin/misk/web/WebActionModule.kt

```diff
@@ -26,7 +27,11 @@ class WebActionModule<A : WebAction> private constructor(
         val provider = getProvider(webActionClass.java)
         @Suppress("UNCHECKED_CAST")
         val binder: Multibinder<BoundAction<A, *>> = Multibinder.newSetBinder(
-                binder(), TypeLiteral.get(Types.newParameterizedType(BoundAction::class.java, Types.subtypeOf(WebAction::class.java), Types.subtypeOf(Object::class.java)))) as Multibinder<BoundAction<A, *>>
+                binder(),
+                TypeLiteral.get(Types.newParameterizedType(BoundAction::class.java,
```

Comment:
we should make some Kotlin sugar for these

Reply:
```
  parameterizedType<BoundAction>(subtypeOf<WebAction>, subtypeOf<Object>).typeLiteral();
```

Reply:
Agreed, I think that's the last "ugly" usage we haven't cleaned up yet. I'll follow up with that in a separate PR tomorrow.